content/docs/futures/spawning.md

```diff
@@ -300,14 +300,12 @@ tokio::run(lazy(|| {
             // Each spawned task will have a clone of the sender handle.
             let tx = tx.clone();
 
-            // In this example, "hello world" will be written to the
-            // socket followed by the socket being closed.
```

Comment:
Instead of removing the comment entirely, should we update it to describe what the code snippet is doing?

Reply:
Something along the lines of `In this example, all bytes read from the socket are placed into a Vec`?

Reply:
That sounds good.